pyproject.toml

```diff
@@ -1,3 +1,90 @@
 [build-system]
-requires = ["setuptools>=41.2", "setuptools_scm", "wheel"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling", "hatch-requirements-txt", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "conda_lock"
+description = "Lockfiles for conda"
+authors = [
+    { "name" = "Marius van Niekerk", email = "marius.v.niekerk@gmail.com" },
+]
+readme = "README.md"
+license = "MIT"
+requires-python = ">=3.6"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Science/Research",
+    "Operating System :: OS Independent",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+]
+dynamic = ["dependencies", "version"]
+license-files = { paths = ["LICENSE"] }
+
+[project.optional-dependencies]
+pip_support = ["poetry ==1.1.*"]
+
+[project.scripts]
+conda-lock = "conda_lock:main"
+
+[project.urls]
+Homepage = "https://github.com/conda-incubator/conda-lock"
+Repository = "https://github.com/conda-incubator/conda-lock"
+Documentation = "https://conda-incubator.github.io/conda-lock/"
+"Issue Tracker" = "https://github.com/conda-incubator/conda-lock/issues"
+PyPI = "https://pypi.org/project/conda-lock/"
```

Comment:
nit: users will only ever be on that page when they see these

```suggestion
```

Reply:
Good point, thanks!